### ISSUE TITLE
Drag and Drop delay

### ISSUE THREAD

**Issue by peasemaker** [yahoo/react-dnd-touch-backend#24]
I want to drag react component when I hold click or touch on it. And when I just touch this component I can scroll the page. How can I do that?

**Comment by hakunin** [comment]
You have to dig into the code and implement it yourself. This repo seems dead and I ended up changing the code to achieve even basic stuff / remove some issues.

**Comment by cedmax** [comment]
There's a delay option I introduced in a previous PR, I probably should have documented it, but it's there 

https://github.com/yahoo/react-dnd-touch-backend/commit/26884a4ae913da3521530f85e6567563a1a8df8e

**Comment by longlho** [comment]
@hakunin sorry for not chiming in (I'm on vacation). I plan to [pull support mouse event](https://github.com/yahoo/react-dnd-touch-backend/issues/26#issuecomment-208098139) in the future (also lack of use cases in our production usage) that's probably why I haven't been responsive to mouse issues

**Comment by oyeanuj** [comment]
@longlho Any updates here on pulling in mouse support like mentioned on your comment above and issue #26?

**Comment by longlho** [comment]
I think I meant pulling out mouse event support, cause it's not entirely 1-1 mapping to HTML5 dnd API and it's buggy for `touch` due to diff propagation scheme

**Comment by oyeanuj** [comment]
@longlho Oh, I see. So, this might be a little bit of an aside, but if one needs to support both touch and mouse events - would the idea be that one uses the html5-backend and this library, switching based on device?

**Comment by longlho** [comment]
yup that's what we've been doing

**Comment by desbrowne** [comment]
@longlho if you pull support for mouse events then what do you suggest for supporting hybrid devices (i.e. MS Surface) where the user may be using both touch and mouse?

**Comment by ehkasper** [comment]
@desbrowne yeah, i've run into the same issue. Any suggestions on how you fixed it (if you did)?

**Comment by desbrowne** [comment]
@ehkasper mouse events haven't been pulled yet, if it does we'll likely have to fork it. So for the time being we simply use touch detection and load the touch backend if it's supported. Older browsers/devices will have a problem with this as touchevent support was buggy but our app requires a modern browser so we eliminate that issue.

**Comment by yummish** [comment]
just to vote for this issue.

We have a responsive web app. It sounds like now we need to switch html5-backend and touch-backend based on touch/mouse. 

I've tried the enableMouseEvents option, but it is quite slow(we have about 100 images). I still need to look into it why it's so slow.

It'd be great to have both support in one single package.

**Comment by hakunin** [comment]
Guys check out this high level DND library:
https://github.com/atlassian/react-beautiful-dnd

Its not a replacement just yet, but I am wathching it closely and will switch soon.

**Comment by dracudakid** [comment]
Try the option `delayTouchStart`
